rollup.config.js

```diff
@@ -0,0 +1,30 @@
+import commonjs from "@rollup/plugin-commonjs"
+import peerDepsExternal from "rollup-plugin-peer-deps-external"
+import postcss from 'rollup-plugin-postcss'
+import resolve from "@rollup/plugin-node-resolve"
+import vue from "rollup-plugin-vue"
+
+import packageJson from "./package.json"
+
+export default {
+  input: "src/index.js",
+  output: [
+    {
+      format: "cjs",
```

Comment:
Why not `umd`?

Reply:
copy pasta from the tutorial. I'm not familiar with current standards. If UMD is the better options I'll happily change to that :)

Reply:
TBH, as this is a private library it doesn't matter too much but in general UMD is more universal since it can be imported anywhere (browsers + nodejs + other bundlers) while CJS can be imported anywhere but browsers. I read somewhere that it's good practice to bundle UMDs when pushing to npm.

However, since modern bundlers will always include the ESM version anyways this is just for legacy reasons :man_shrugging:

Reply:
So all in all I'd say keep it as `cjs`.